libraries/AP_HAL_ChibiOS/hwdef/VUAV-V7pro/README.md

```diff
@@ -0,0 +1,607 @@
+# VUAV-V7pro Flight Controller
+
+The VUAV-V7pro  flight controller is manufactured and sold by [VIEWPRO](http://www.viewprotech.com/).
+
+## Features
+
+ - STM32H743 microcontroller
+ - Three IMUs: ADIS16470,ICM42688,ICM42688
+ - Internal vibration isolation for IMUs
+ - Internal RM3100 SPI magnetometer
+ - Internal two MS5611 SPI barometer
+ - Internal RGB LED
+ - MicroSD card slot port
+ - 1 Analog power port
+ - 1 CAN power port
+ - 5 UARTs and 1 USB ports
+ - 1 RS232 port
+ - 14 PWM output ports
+ - 4 I2C and 2 CAN ports
+ - Safety switch port
+ - Buzzer port
+ - RC IN port
+
+##  Pinout
+
+![VUAV-V7pro-interface.png](VUAV-V7pro-interface.png)
+
+![VUAV-V7pro-pinout.png](VUAV-V7pro-pinout.png)
+
+## UART Mapping
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART2 (Telem1)
+ - SERIAL2 -> UART6 (Telem2)

```

Comment:
```suggestion
 - SERIAL2 -> UART6 (Telem2)  (DMA enabled)

```

Reply:
Okay, I'll modify it and then force the push